### PR TITLE
Chore/workdir

### LINKED ISSUE #23: Chore: Support for Current Working Directory
# Feature Description: 

when users use Cogo Client it communicate to the Cogo daemon and request command execution, but how does the daemon know in what **Context** to execute requested commands?

what will be the results of this commands?:
```Console
~/private/b $ cogo -s dir_a run ls -la
```
```Console
~/work/a    $ cogo -s dir_b run ls -la
```

# Expected Behavior:

- cogo client should require the use of **Context** 
- **Context** will include a $cwd adding the location the client was invoked from
- support running commands in multiple location for daemon

### PR DESCRIPTION
**Pull request by yavidor**
Fixes #23

### CODE REVIEW

File: common/workdir.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package common
+
+import (
+	"os"
+	"path/filepath"
+)
+
+func JoinWithWorkDir(paths ...string) string {
```
Comment (royiro10) [inline common/workdir.go]: great, this will allow us to always reference files based on relative position to source code dir

Comment (royiro10) [reply common/workdir.go]: but I think the name `JoinWithWorkDir` is a bit unclear

Comment (yavidor) [reply common/workdir.go]: I agree with you, no idea how to name it

**Comment by yavidor** [comment]
I am afraid the workdir is being passed around to much between functions, no real idea how to fix it.

**Pull request merged**